origin_one_description/urdf/README.md

```diff
@@ -40,7 +40,7 @@ Or build your own robot with this robot as a base. E.g. if you want to add a new
 <model name="modified_origin">
     <include>
     <uri>
-        https://fuel.gazebosim.org/1.0/AvularRobotics/models/AvularOriginv10
+        https://fuel.gazebosim.org/1.0/AvularRobotics/models/AvularOriginone
```

Comment:
Did we actually put it on fuel already? If so, shouldn't it be AvularOriginOne?

Reply:
I did not put anything on fuel, i leave that to you ;) Will change the text